### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #9 [April] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9365] [Wallet and (d)App Council]
### Join
Google Meet: [meet.google.com/uwj-wfse-uzm](https://meet.google.com/uwj-wfse-uzm)
Recording: See in comments👇

### Useful links

- Notion page with meeting notes: [Celo Wallet Council Meeting Notes](https://www.notion.so/celoplatform/Celo-Wallet-Council-61dfe28e98a543409cf6c181b132c4de)
- Last meeting: https://github.com/celo-org/celo-monorepo/discussions/9304
- Forum post: https://forum.celo.org/t/celo-wallet-and-d-app-council-9-april/3179/2
--------------------------------------------

Hey all :wave:

The monthly Celo Wallet and (d)App Council meeting is taking place on **Wednesday, March 30th (9am PDT/6pm CEST)**.

### :calendar: What is this?

- A *monthly* meeting for Celo **developers, founders,** **product** people (anyone really!) to share
  - wishes,
  - pain points, and
  - ideas

### 🗒️  Agenda

- Check out, vote, and add to the agenda below
- 🗳️ You can **upvote/downvote** agenda items below.
- ✍️ Please feel free to add agenda items (however small or big!) on Github. For example:

  - broad discussion topics to **amplify** your ideas/features/frictions,
  - requests for **feedback** (from the community),
  - roadmap **updates** ,
  - etc (anything that would be useful for you really!)

### :star_struck: How do I participate?

- Join using Google Meet: meet.google.com/uwj-wfse-uzm
- Take a peek at the [previous meeting notes](https://www.notion.so/celoplatform/Celo-Wallet-and-d-App-Council-monthly-61dfe28e98a543409cf6c181b132c4de)

See you there! :rocket:

--------------------------------------------

### Housekeeping:

- [x] Send update calendar invite with Github link
- [x] Add agenda items to vote on
- [x] Publicise on forum and Discord

--------------------------------------------

### Proposed agenda items 👇  

Please feel free to add agenda items (however small or big!) below.
 
For example:
- broad discussion topics to **amplify** your ideas/features/frictions, 
- requests for **feedback** (from the community), 
- roadmap **updates**,
- etc (anything that would be useful for you really!) 

🗳️ You can upvote/downvote agenda items below.

**Comment by aaronmgdr** [comment]
ContractKit 2.0 (What changes and what we get)

**Reply by arthurgousset** [reply]
For context: 
- https://github.com/celo-org/celo-monorepo/discussions/9364

**Reply by aaronmgdr** [reply]
Migration Guide (out a little early) 

https://docs.celo.org/developer-guide/contractkit/migrating-to-contractkit-v2

**Comment by aaronmgdr** [comment]
the `use-contrackit` `dappkit` `valora` trilemma 

Getting to a solution where there is a good experience both mobile to mobile and mobile to deskop for users and developers celo based dApps.

**Comment by albertomartin** [comment]
Proposed subject: Payments SDK

**Comment by priyank-makerdojo** [comment]
Proposed Subject: Get feedback on the newly built Celo Analytics tool, similar to Dune.

Who we are: We built a tool similar to Dune for Celo. Using it, you can create SQL queries, create dashboards, share them publicly, etc., 

Goal: We would like to show what we built and hear from the fellow Celo community about their data needs so we can make specific improvements in that direction

Here is an example dashboard on Celo transactions 
<img width="1678" alt="image" src="https://user-images.githubusercontent.com/101025565/160669382-9c9258db-b6eb-4cd7-b399-91adb95ab682.png">

**Comment by arthurgousset** [comment]
Thanks to all who attended the recent Wallet and (d)App Council meeting. 

Here is the:

- Meeting recording 🎥: https://drive.google.com/file/d/1Litt2SZgRWzFtPEBvX00oDpfz2Fp3D9d/view
- Meeting notes 📝: https://www.notion.so/celoplatform/Celo-Wallet-and-d-App-Council-monthly-61dfe28e98a543409cf6c181b132c4de#ac7b0a87fc4544e8882ceb649aee5448
